src/Tools/Md2Html/Program.cs

```diff
@@ -0,0 +1,64 @@
+﻿using System;
+using System.IO;
+
+namespace Md2Html
+{
+    static class Program
+    {
+        static void Main(string[] args)
```

Comment:
have you considered using ndesk options in the repo to list CLI options / help docs

Reply:
Nop, but I will take a look.

Reply:
Added basic help